src/test/kfs_rest_test.cpp

```diff
@@ -154,6 +194,73 @@ class HttpRestApiHandlerWithDynamicModelTest : public HttpRestApiHandlerTest {
 std::unique_ptr<MockedServer> HttpRestApiHandlerTest::server = nullptr;
 std::unique_ptr<std::thread> HttpRestApiHandlerTest::thread = nullptr;
 
+void testInference(int headerLength, std::string& request_body, std::unique_ptr<HttpRestApiHandler>& handler) {
+    std::string request = "/v2/models/mediapipeAdd/versions/1/infer";
+
+    std::vector<std::pair<std::string, std::string>> headers;
+    std::pair<std::string, std::string> binaryInputsHeader{"Inference-Header-Content-Length", std::to_string(headerLength)};
+    headers.emplace_back(binaryInputsHeader);
+
+    ovms::HttpRequestComponents comp;
+
+    ASSERT_EQ(handler->parseRequestComponents(comp, "POST", request, headers), ovms::StatusCode::OK);
+
+    std::string response;
+    ovms::HttpResponseComponents responseComponents;
+    ASSERT_EQ(handler->dispatchToProcessor(request_body, &response, comp, responseComponents), ovms::StatusCode::OK);
+
+    std::cout << "RESPONSE" << response.c_str() << std::endl;
```

Comment:
Isn't it too big? I wonder if we should print this part.

Reply:
I will remove this debug line.